app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt

```diff
@@ -165,6 +167,7 @@ class BrokenSiteActivity : DuckDuckGoActivity() {
         private const val CONSENT_MANAGED_EXTRA = "CONSENT_MANAGED_EXTRA"
         private const val CONSENT_OPT_OUT_FAILED_EXTRA = "CONSENT_OPT_OUT_FAILED_EXTRA"
         private const val CONSENT_SELF_TEST_FAILED_EXTRA = "CONSENT_SELF_TEST_FAILED_EXTRA"
+        private const val BOOLEAN_PARAMS = "BOOLEAN_PARAMS"
```

Comment:
sending a list of params here that will end up like 

"<param>=true"

Does this way because this data is send through the Intent and I didn't know how to serialize a map through it :)